### PR TITLE
Fix GitHub check issues by updating actions to v2

### PR DESCRIPTION
**Pull request by torfason**
I noticed that after my changes were merged in, `R-CMD-check` on GitHub started failing. I went exploring and it looks like it did not have to do with my commits, but that the actions/workflows were not working anymore. I've had this problem with different packages, it seems like old workflows don't work in many cases.

I believe the solution is to update the workflows to use `v2` of the `r-lib` actions. I did this by simply deleting the old actions and creating new ones using the `usethis` methods, as suggested in the release notes for `r-lib/actions`. This includes a bit fewer platforms than the original, but I would think it's still enough to provide quite good reassurance:

* https://www.tidyverse.org/blog/2022/06/actions-2-0-0/

This PR includes those changes, and adds the word `CMD` to the `WORDLIST`, which seems to have been causing a note.

This version passes all tests in my local repo (https://github.com/torfason/mustashe/actions), and here on the PR.

### CODE REVIEW

**Review by jhrcook — APPROVED** [review]
Everything looks good to me 👍🏼

File: .github/workflows/check-standard.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
```
Comment (jhrcook) [inline .github/workflows/check-standard.yaml]: Can you remove the `main` branch option here.

Comment (torfason) [reply .github/workflows/check-standard.yaml]: This should be fixed now.

**Review by jhrcook — APPROVED** [review]
Good to merge.

**Pull request merged**